### PR TITLE
README.md Update

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Check List (Check all the applicable boxes)

- [ x ] My code follows the code style of this project.
- [ x ] My change requires changes to the documentation.
- [ x ] I have updated the documentation accordingly.
- [ x ] All new and existing tests passed.
- [ x ] This PR does not contain plagiarized content.
- [ x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1613"></a>

### CODE REVIEW

**Review by CodingSpecies — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Before contributing or adding a new feature, please make sure you have already i
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```
Comment (Panquesito7) [inline README.md:40]:
```suggestion
2. Clone the project by running `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Before contributing or adding a new feature, please make sure you have already i
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project by running `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```
Comment (Panquesito7) [inline README.md:40]: Now that I think about it, the fork is the repository the user must clone, not the original repository.
```suggestion
2. Clone the project by running `git clone https://github.com/<your-username>/LinkFree.git`
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Comment (CodingSpecies) [reply README.md:40]: Oh sorry, I reviewed without noticing this!

Comment (kumarsonsoff3) [reply README.md:40]: No worries @CodingSpecies, it does happen! We all learn from mistakes! 😄

Comment (kumarsonsoff3) [reply README.md:40]: May I know, why you are **marking this conversation resolved** again and again **without making changes**? We'd love to know your Point of view!! 

I think these changes should be made, to be very specific, especially for the first timers!! 
What do you think!?

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**